requestlogs/entries.py

```diff
@@ -111,14 +111,16 @@ def store(self):
     @property
     def user(self):
         ret = {
-            'id': None,
-            'username': None,
```

Comment:
The `'username'` still has to be in the returned dictionary, because the default user serializer (`BaseEntrySerializer -> UserSerializer`) expects it (otherwise tests will fail, for example).

Reply:
Added it myself here https://github.com/Raekkeri/django-requestlogs/pull/32

Reply:
Thanks @Raekkeri